share.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Share
+excerpt: "Do you like ev3dev? Show your support! We invite you to star the repo on GitHub and submit your cool projects to our projects page."
```

Comment:
This isn't the best choice for an excerpt. It actually would be good as an alert on this page. The excerpt should say something to the effect that this page lets you find project that other have shared and tells you how to share your own projects.